deployment/network-operator/values.yaml

```diff
@@ -225,6 +225,15 @@ ibKubernetes:
   pKeyGUIDPoolRangeEnd: "02:FF:FF:FF:FF:FF:FF:FF"
   ufmSecret: # specify the secret name here
 
+nvIpam:
+  deploy: false
+  image: nvidia-k8s-ipam
+  repository: ghcr.io/mellanox
+  version: latest
```

Comment:
do we really whant to use `latest` tag? we don't use it with other components

Reply:
we do :), will update to v0.0.2